content/en/includes/task-tutorial-prereqs.md

```diff
@@ -1,8 +1,5 @@
-You need to have a Kubernetes cluster, and the kubectl command-line tool must
-be configured to communicate with your cluster. It is recommended to run this tutorial on a cluster with at least two nodes that are not acting as control plane hosts. If you do not already have a
-cluster, you can create one by using
-[minikube](https://minikube.sigs.k8s.io/docs/tutorials/multi_node/)
-or you can use one of these Kubernetes playgrounds:
+- If you do not already have a cluster, you can create one by using [minikube](https://minikube.sigs.k8s.io/docs/tutorials/multi_node/) or you can use one of these Kubernetes playgrounds:
+    - [Killercoda](https://killercoda.com/playgrounds/scenario/kubernetes)
+    - [Play with Kubernetes](https://labs.play-with-k8s.com/)
 
-* [Killercoda](https://killercoda.com/playgrounds/scenario/kubernetes)
-* [Play with Kubernetes](https://labs.play-with-k8s.com/)
+* Understand how to [configure a Pod to use a ConfigMap](/docs/tasks/configure-pod-container/configure-pod-configmap/).
```

Comment:
I suggest you to preserve the `task-tutorial-prereqs.md` as-is, not modifying in this PR.

Reply:
The formatting in the current task tutorial file is not usable. It has to be changed. Who's approval do we need to get it changed?

Reply:
Hi Jerry, you should clarify how it's not usable in your PR description and why the change is important.

Reply:
Existing formatting seems fine to me, so an explanation of why you find it not usable would be helpful.